style.css

```diff
@@ -0,0 +1,78 @@
+body {
+  display: flex;
+  flex-direction: column;
+  height: 700px;
+  background-color: #1c1a19;
+  background-image: url("Illustration.png");
+  background-repeat: no-repeat;
+  background-position-x: right;
+  background-position-y: 45vh;
+}
+
+.toolbar {
+  display: flex;
+  justify-content: space-between;
+  align-items: baseline;
+  padding-left: 20px;
+  padding-right: 20px;
+  margin: -10px;
+  background-color: #3c3a39;
+}
+
+nav a {
+  color: #fff;
+}
+
+.toolbar a {
+  text-decoration: none;
+  color: #fff;
+  font-family: Roboto, BlinkMacSystemFont, 'Segoe UI', Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
+}
+
+nav a:hover {
+  color: turquoise;
+}
+
+.social li a {
+  color: #dbd8d7;
+}
+
+.toolbar a:hover {
+  color: turquoise;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  padding: 65px 20px;
+}
+
+.card h1 {
+  font-size: 56px;
+  color: #ff6b00;
+  font-family: Crete Round, sans-serif, 'Times New Roman', Times, serif;
+}
+
+.card h3 {
+  font-size: 24px;
+  color: #fff5e1;
+  font-family: Roboto, BlinkMacSystemFont, 'Segoe UI', Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
+}
+
+.card p {
+  font-size: 16px;
+  color: #fff;
+  font-family: Poppins, sans-serif, 'Times New Roman', Times, serif;
+}

```

Comment:
- [ ] **OPTIONAL** In the styling, please notice that you should follow the exact stylings as the given design template in Figma in order for you to practice following the guidelines of a given template and for the website to be professional, consistent, and good-looking.

> Keep in mind that you should follow the given design in typography, font size, font weight, colors, images, spacings (margin and padding), and button interactions such as hover, active (when it's pressed), disabled, and enable (default state).

_Kindly look at the images down below, then apply the corresponding changes._

### **Your design**
![Screenshot1](https://user-images.githubusercontent.com/47625407/121258417-29bcc080-c8af-11eb-95dc-5fd79e22ab70.png)
### **[Original design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471)**
![Screenshot](https://user-images.githubusercontent.com/47625407/121258464-36411900-c8af-11eb-97a5-32bc2673be74.png)

> Please note that this is a mandatory requirement, and I'm sorry that it didn't show up for you in the earlier review.